tests/unittests/tests-gnrc_sixlowpan_frag_vrb/tests-gnrc_sixlowpan_frag_vrb.c

```diff
@@ -58,14 +57,13 @@ static uint8_t _out_dst[] = TEST_OUT_DST;
 static void set_up(void)
 {
     gnrc_sixlowpan_frag_vrb_reset();
-    tag = 0;
```

Comment:
I am browsing through the test applications first. Was the removal of the `tag test` intended? If yes, why?

Reply:
> Was the removal of the `tag test` intended? If yes, why?

This change to the test application is part of https://github.com/RIOT-OS/RIOT/pull/12559. With that PR the tag generator `gnrc_sixlowpan_frag_next_tag()` becomes independent of the `gnrc_sixlowpan_frag` module, so it does not need to be re-implemented, whenever the VRB is used without `gnrc_sixlowpan_frag` (as e.g. in this test or in SFR). The caveat is, that these tests don't know what the last tag was (they could assume it is just incremented, but it they shouldn't as this isn't necessarily future proof). Let's discuss this further in #12559 ;-).

> I am browsing through the test applications first. 

As stated in OP: this PR does not contain any test applications yet. All test applications or changes to test applications seen here are part of depending PRs.